src/parameter_bridge.cpp

```diff
@@ -154,15 +174,34 @@ int main(int argc, char * argv[])
   {
     for (size_t i = 0; i < static_cast<size_t>(services_2_to_1.size()); ++i) {
       std::string service_name = static_cast<std::string>(services_2_to_1[i]["service"]);
-      std::string package_name = static_cast<std::string>(services_2_to_1[i]["package"]);
       std::string type_name = static_cast<std::string>(services_2_to_1[i]["type"]);
+      {
+        // for backward compatibility
+        std::string package_name = static_cast<std::string>(services_2_to_1[i]["package"]);
+        if (!package_name.empty()) {
+          fprintf(
+            stderr,
+            "The service '%s' uses the key 'package' which is deprecated for "
+            "services. Instead prepend the 'type' value with '<package>/'.\n",
+            service_name.c_str());
+          type_name = package_name + "/" + type_name;
+        }
+      }
       printf(
-        "Trying to create bridge for ROS 1 service '%s' "
-        "with package '%s' and type '%s'\n",
-        service_name.c_str(), package_name.c_str(), type_name.c_str());
+        "Trying to create bridge for ROS 1 service '%s' with type '%s'\n",
+        service_name.c_str(), type_name.c_str());
+
+      size_t index = type_name.find("/");
```

Comment:
@dirk-thomas nit^N: can be made `const`

Reply:
Done in 5ea5a40c23a65cef995caba7d8334ed482d65c8f.